src/main/java/org/privacyidea/JSONParser.java

```diff
@@ -174,6 +174,15 @@ else if ("interactive".equals(modeFromResponse))
             response.type = getString(detail, TYPE);
             response.otpLength = getInt(detail, OTPLEN);
 
+            String r = getString(detail, CHALLENGE_STATUS);
+            for (ChallengeStatus en : ChallengeStatus.values())
```

Comment:
why call it `en`? could just be `cs`. also formatting, no whitespace after `en`

Reply:
[5f48ebd](https://github.com/privacyidea/java-client/pull/68/commits/5f48ebd9138f48d5e203537fe3a660b11449d42f)